osquery/sql/sqlite_util.cpp

```diff
@@ -459,40 +461,85 @@ Status QueryPlanner::applyTypes(TableColumns& columns) {
   return Status(0);
 }
 
-int queryDataCallback(void* argument, int argc, char* argv[], char* column[]) {
-  if (argument == nullptr) {
-    VLOG(1) << "Query execution failed: received a bad callback argument";
-    return SQLITE_MISUSE;
+Status osquery_exec(
```

Comment:
sqlite3_exec handles multi-statement queries.  Unless I'm mistaken, this does not.

Reply:
Thanks for the feedback. Yes that was my intentional decision, this sqlite3_exec was not used for multiple query execution and also should not. When several similar tables are appended union is used, when tables are not similar it means we are parsing and sending a lot of null entries. Thats why i decided to limit this capability in the new function.

Reply:
How can one know that it was never used for multi-statement queries?  In fact, (again, unless I'm mistaken) we have multi-statement queries in our own internal use of osquery.